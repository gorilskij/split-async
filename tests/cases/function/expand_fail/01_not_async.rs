#![no_main]

use split_async::split;

#[split]
fn func() {}
