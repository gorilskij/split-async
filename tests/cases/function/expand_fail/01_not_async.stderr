error: function must be async
 --> tests/cases/function/expand_fail/01_not_async.rs:6:1
  |
6 | fn func() {}
  | ^^^^^^^^^
